mewarpx/mewarpx/utils_store/init_restart_util.py

```diff
@@ -0,0 +1,52 @@
+"""
+Utility functions to start a run from a checkpoint or restart.
+"""
+import os
+import shutil
+import logging
+import traceback
+
+from mewarpx.mwxrun import mwxrun
+
+logger = logging.getLogger(__name__)
+
+
+def clean_old_checkpoints(prefix):
+    if os.path.isdir("diags"):
+        for d in next(os.walk("diags"))[1]:
+            if d.startswith(prefix):
+                print(f"Removing old checkpoint file {d}")
+                shutil.rmtree(os.path.join("diags", d))
+
+def run_restart(checkpoint_prefix, force=False):
+
+    logger.info(f"Attempting to " + ("force a " if force else "") + "restart from the most recent checkpoint starting with '{checkpoint_prefix}'")
+
+    if not os.path.isdir("diags"):
+        if force:
+            raise RuntimeError("Diags directory does not exist!")
+        else:
+            logger.warning("Diags directory does not exist!")
+            return
```

Comment:
I would `return False` so that if the user wants they can check if the restart was successful or not based on this function's return.